### PR TITLE
Adding new api version 2020-06-01 for compute

### PR DESCRIPTION
**Pull request by sirajudeens**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/compute/resource-manager/Microsoft.Compute/stable/2020-05-01/compute.json [hunk 1]
```diff
@@ -0,0 +1,10988 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ComputeManagementClient",
+    "description": "The Compute Management Client.",
+    "version": "2019-12-01"
```
Comment (ravbhatnagar) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2020-05-01/compute.json:6]: this will also need to change, is that a miss?

**Review by ravbhatnagar — COMMENTED** [review]
This PR is adding new properties to existing resource model. Its fine since its new API-version. ARM can sign off, just one comment to fix. Not sure if this was a miss or deliberate?

**Review by hyonholee — COMMENTED** [review]
@smotwani Please confirm that all new features of CRP for 2020-05-01 API version are included in this PR.  If not, send a PR to sirajudeens:new-api-version branch and update the branch before merging this PR.

File: specification/compute/resource-manager/Microsoft.Compute/stable/2020-05-01/compute.json (hunk 1 above)
Comment (sirajudeens) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2020-05-01/compute.json]: Yes. This is fixed now.

**Comment by sirajudeens** [comment]
This is no more required.

**Pull request closed without merging**